File: tests/fixtures/sample.java
package com.example;

interface Drawable {
    void draw();
}

enum Status {
    ACTIVE,
    INACTIVE
}

public class Shape implements Drawable {
    private String name;

    public Shape(String name) {
        this.name = name;
    }

    @Override
    public void draw() {
        System.out.println("Drawing " + name);
    }
}

File: tests/fixtures/sample.kt
package com.example

interface Drawable {
    fun draw()
}

enum class Status {
    ACTIVE,
    INACTIVE
}

class Shape(private val name: String) : Drawable {
    override fun draw() {
        println("Drawing $name")
    }
}

object Utils {
    fun add(a: Int, b: Int): Int = a + b
}

fun greet(name: String): String = "Hello, $name"
